### PR TITLE
USAGOV-1942-icon-all-benefits

### PR DESCRIPTION
**Pull request by omerida**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1942

## Description

Update CSS to use life event ID in CSS class instead of title. Adds a selector to display icons for an "All Benefits" life event

## Type of Changes

- [ ] New Feature
- [ ] Bugfix
- [x] Frontend (Twig, Sass, JS)

image

- [x] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] Egress
  - [ ] Tools
- [ ] Other

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [x] Requires New Config
- [] Requires New Content

### Validation Steps

1. Import configuration to update life events grid view
2. Create or publish a life event with ID="all_benefits", with a search title and short description field. Language=en
3. Create or publish a life event with ID="es_all_benefits", with a search title and short description field. Language=es
4. On English page, `/beneft-finder' page, confirm a new tile displays in the grid with an icon and the title and short description from step 2 above.
5.  On Spanish page, `/es/buscador-beneficios' page, confirm a new tile displays in the grid with an icon and the title and short description from step 3 above.

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
6. Find the commit of this pull request.
7. Build and deploy the changes.
8. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

**Comment by IsabelLaurenceau** [comment]
The Card is coming up but the icon is different from the one you have shown above. On the English page the Spanish card is also coming up. 
 
Screenshot 2024-09-19 at 9 14 32 AM
Screenshot 2024-09-19 at 9 14 58 AM

**Comment by omerida** [comment]
ah, I think Gene / Scott changed it. I'll take a look

**Comment by omerida** [comment]
The icon coming up is correct, I must have grabbed that screenshot with a cached version of the icon or the wrong path in CSS. The one is the sass file is correct, the person with a circle of arrows around them.

I can't replicate the Spanish showing up on the English page. I copied the content as it is on cms.usa.gov for the two new life events on my local. Can you check the language of the Spanish "all benefits" card?

**Comment by IsabelLaurenceau** [comment]
@omerida let me try again!

**Comment by YaritzaGarcia** [comment]
It's working for me! 
Screenshot 2024-09-20 at 1 32 15 PM
Screenshot 2024-09-20 at 1 32 29 PM

**Comment by IsabelLaurenceau** [comment]
Same! I'm approving now :)

**Review by IsabelLaurenceau — APPROVED** [review]
